src/Psl/Str/convert_encoding.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Str;
+
+use function mb_convert_encoding;
+
+/**
+ * Detect the encoding of the giving string.
+ *
+ * @psalm-return null|string The string encoding or null if unable to detect encoding.
+ *
+ * @psalm-pure
+ */
+function convert_encoding(string $string, string $from, string $to): ?string
+{
+    return mb_convert_encoding($string, $to, $from) ?: null;
```

Comment:
`mb_convert_encoding` always returns a string or an array. and since we are not passing an array, its always gonna be return a string, so `?: null` can be removed, and the return type should be changed to `string`